app/org/sagebionetworks/bridge/migration/HealthDataDumper.java

```diff
@@ -0,0 +1,95 @@
+package org.sagebionetworks.bridge.migration;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import java.io.BufferedWriter;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+import java.io.Writer;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+
+import org.joda.time.DateTime;
+import org.sagebionetworks.bridge.config.Environment;
+import org.sagebionetworks.bridge.dao.HealthDataDao;
+import org.sagebionetworks.bridge.models.healthdata.HealthDataRecord;
+import org.springframework.context.support.AbstractApplicationContext;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+class HealthDataDumper {
+
+    public static void main(String[] args) throws FileNotFoundException, IOException {
+        final AbstractApplicationContext appContext = Utils.loadAppContext();
+        HealthDataDumper dataDumper = new HealthDataDumper(Environment.UAT, "heroku", appContext);
+        DateTime start = new DateTime(2015, 7, 7, 0, 0);
+        DateTime end = new DateTime(2015, 7, 10, 0, 0);
```

Comment:
I assume these are test dates, because the problem started happening in early June.